src/NullBackupProfile.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace MWStake\MediaWiki\CliAdm;
+
+class NullBackupProfile implements IBackupProfile {
```

Comment:
Looks like this can be solved by just inheriting from JsonBackupProfile and overriding the default constructor.

Also, maybe it would be a good idea to actually invert the inheritance then and make JsonBackupProfile inherit from the "DefaultBackupProfile"

Reply:
You're right, did it that way.